extensions/arc/package.nls.json

```diff
@@ -81,10 +81,14 @@
 	"arc.azure.section.title": "Azure information",
 	"arc.sql.instance.name": "Instance name (lower case letters and digits only)",
 	"arc.sql.username": "Username",
-	"arc.sql.storage-class.data.label": "Storage Class (Data)",
+	"arc.storage-class.data.label": "Storage Class (Data)",
 	"arc.sql.storage-class.data.description": "The storage classes to be used for data (.mdf)",
-	"arc.sql.storage-class.logs.label": "Storage Class (Logs)",
+	"arc.postgres.storage-class.data.description": "The storage classes to be used for data persistent volumes",
```

Comment:
It's a single storage class being provided, so I would say "The storage class" instead of "The storage classes" for these 3.